lerna.json

```diff
@@ -1,11 +1,14 @@
 {
   "lerna": "2.4.0",
-  "version": "0.0.0",
+  "version": "0.7.0",
```

Comment:
Based on discussions with Marc, we should keep the version as `0.0.0`
(and align the `cpp-debug` and `cortex-debug` to `0.0.0`)
This means that when we publish (the first latest), `lerna` will step the versions properly at `0.1.0`.

cc @marcdumais-work